db/fixtures/miq_user_roles.yml

```diff
@@ -33,6 +33,8 @@
   - ems_cluster
   - ems_infra
   - ems_physical_infra
+  - event_stream_show_list
+  - event_stream
```

Comment:
I think this is fine. I followed the pattern of other api's that do not use _show.